x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/execution_enumeration_via_wmiprvse.json

```diff
@@ -0,0 +1,45 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies native Windows host and network enumeration commands spawned by the Windows Management Instrumentation Provider Service (WMIPrvSE).",
+  "from": "now-9m",
+  "index": [
+    "winlogbeat-*",
+    "logs-endpoint.events.*",
+    "logs-windows.*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Enumeration Command Spawned via WMIPrvSE",
+  "query": "process where event.type in (\"start\", \"process_started\") and\n  process.name:\n  (\n    \"arp.exe\",\n    \"dsquery.exe\",\n    \"dsget.exe\",\n    \"gpresult.exe\",\n    \"hostname.exe\",\n    \"ipconfig.exe\",\n    \"nbtstat.exe\",\n    \"net.exe\",\n    \"net1.exe\",\n    \"netsh.exe\",\n    \"netstat.exe\",\n    \"nltest.exe\",\n    \"ping.exe\",\n    \"qprocess.exe\",\n    \"quser.exe\",\n    \"qwinsta.exe\",\n    \"reg.exe\",\n    \"sc.exe\",\n    \"systeminfo.exe\",\n    \"tasklist.exe\",\n    \"tracert.exe\",\n    \"whoami.exe\"\n  ) and\n  process.parent.name:\"wmiprvse.exe\"\n",
```

Comment:
I think when this happens it's typically like `cmd /c disc,overycommands.bat > c:\something.txt` which this rule would miss.

the reason for that is because there's no way to get stdout when launching a process via wmiprvse.exe, so you often see it redirected to a file.

so for the above examples, there's no way for anyone to get the output of those commands, so even though those events could happen, they are useless.

an alternative approach:
- look for `cmd /c > * .txt` as a child command


there could be noise (I've seen nessusd do this), but it should be all automated processes and easy for users to add exceptions

Reply:
After discussing with @bm11100, this is definitely a good approach, but different enough to justify it being a different rule. The rule in its current state would be difficult to capture the output, but that may not be the intent. I will make an issue to create a rule similar to your approach for a future release